### ISSUE TITLE
rclone create/connect repository error 

### ISSUE THREAD

**Issue by rambalachandran** [kopia/kopia#2671]
I use kopia-ui to connect to a  repository through rclone back-end (to google drive). Recently I modified my machine a bit and when I try to reconnect to the repository, I get the following error

```
Connect Error: INTERNAL: internal server error: connect error: unable to write blobcfg blob: PutBlob() failed for "kopia.blobcfg": error writing blob using WebDAV: unable to complete WriteTemporaryFileAndCreateParentDirs despite 10 retries: BLOB not found
```
Tried to uninstall kopia and reinstall. Also tried to downgrade (0.10.7) and still facing the same issue. I also face the same issue when I try to create a new repository in the same backend.

I'm able to access the files using` rclone ls` or `rclone lsd`. I'm also able to edit the backend to add folders using `rclone mkdir`. Only kopia is unable to use the rclone backend

**Comment by basldfalksjdf** [comment]
Are you on v1.61 of rclone? Then this may be your issue: https://github.com/kopia/kopia/issues/2655

**Comment by rambalachandran** [comment]
Ah yes I'm . I just updated. Let me try to downgrade and test

**Comment by metabubble** [comment]
I am using the docker and am affected. Will a new docker image be provided to fix this?

**Comment by rambalachandran** [comment]
Can confirm the error is with rclone v1.61. I downgraded to v1.60 and it works fine

**Comment by revilo951** [comment]
I'm in the same boat as metabubble - using docker, and this is causing issues - is there a plan to sort this for docker?

**Comment by jkowalski** [comment]
Which docker tag are you using?

**Comment by metabubble** [comment]
> Which docker tag are you using?

latest

**Comment by jkowalski** [comment]
Strange. Kopia ships with rclone 1.59 inside the docker image. Can you double check the version in the image?

**Comment by metabubble** [comment]
It is indeed 1.59

`root@xxxx /var/lib/docker/overlay2 # ./c999ce0exxxx/merged/usr/bin/rclone --version
rclone v1.59.0
- os/version: debian 11.5 (64 bit)
- os/kernel: 5.10.0-19-amd64 (x86_64)
- os/type: linux
- os/arch: amd64
- go/version: go1.18.3
- go/linking: static
- go/tags: none
`

I am an idiot :D

First part of the message was the same, bit in garbled, urlencoded text it said I need to reconnect my rclone remote. As far as I am concerned, issue is unrelated. Thanks for your help!

**Comment by lupusA** [comment]
Should be fixed by #2655 as it was reported to work with rclone v1.61.
